src/model/demonlist/demon.rs

```diff
@@ -212,7 +212,21 @@ impl Demon {
     }
 
     pub fn score(&self, progress: i16) -> f64 {
-        let beaten_score = 150f64 * f64::exp((1f64 - f64::from(self.base.position)) * (1f64 / 30f64).ln() / (-149f64));
+        let position = self.base.position as f64;
+        if 125 < position && position >= 150: 
+            let beaten_score = 150f64 * f64::exp((1f64 - f64::from(self.base.position)) * (1f64 / 30f64).ln() / (-149f64));
+        else if 50 < position && position <= 125:
+            let a = 2.333f64;
+            let b = 1.884f64;
+            let beaten_score = 60f64 * (a.powf((51f64 - position) * ((30f64) / 99f64).ln())) + b;
+        else if 20 < position && position <= 50:
+            let c = 1.01327f64
+            let d = 26.489f64
+            let beaten_score = -100f64 * (c.powf(placement - d)) + 200f64
+        else if 0 < placement <= 20:
+            let e = 1.168f64
+            let f = 100.39f64
+            let beaten_score = (250f64 - f) * (e.powf(1f64 - placement)) + f
```

Comment:
this is python syntax. In rust all blocks need to be enclosed by curly braces. If-statements aren't followed by colons. Each line has to be terminated by a semi-colon (unless the return value, but none of the lines here are).